OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseTabsDashboardFragment.java

```diff
@@ -135,20 +134,21 @@ public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
             environment.getAnalyticsRegistry().trackScreenView(
                     Analytics.Screens.COURSE_DASHBOARD, courseData.getCourse().getId(), null);
             if (!courseData.getCourse().getCoursewareAccess().hasAccess()) {
+                setupToolbar(false);
                 final boolean auditAccessExpired = new Date().after(DateUtil.convertToDate(courseData.getAuditAccessExpires()));
                 if (auditAccessExpired) {
                     if (courseData.isUpgradeable() && environment.getAppFeaturesPrefs().isValuePropEnabled()) {
                         setupIAPLayout();
                     } else {
-                        setupToolbar(false);
                         binding.accessError.setVisibility(View.VISIBLE);
-                        binding.accessError.setState(EdxCourseAccessErrorState.State.AUDIT_ACCESS_EXPIRED);
+                        binding.accessError.setState(EdxCourseAccessErrorState.State.AUDIT_ACCESS_EXPIRED, null);
                         binding.accessError.setPrimaryButtonListener(onFindCourseClick());
                     }
                 } else {
-                    FragmentDashboardErrorLayoutBinding errorLayoutBinding = FragmentDashboardErrorLayoutBinding.inflate(inflater, container, false);
-                    errorLayoutBinding.errorMsg.setText(R.string.course_not_started);
-                    return errorLayoutBinding.getRoot();
+                    binding.accessError.setVisibility(View.VISIBLE);
```

Comment:
I guess we can't remove the `FragmentDashboardErrorLayoutBinding` as for now because currently we are only handling the case in which the course data has the started date other cases are still remaining as per [figma](https://www.figma.com/file/DWFQZ9LNGdu2K3ZBUN7o9a/Access-expired-message?node-id=6%3A791&t=WLmTgeLyzHG3WfD8-0) designs

Reply:
Added a Todo as well.